frontend/apps/web/src/pages/system/_name.tsx

```diff
@@ -28,95 +33,224 @@ const RepositoryDetails = () => {
   const totalTests = system.subsystems.reduce((totalTests, subsystem) => {
     return totalTests + subsystem.tests.length
   }, 0)
+  const navigate = useNavigate()
+  const [activeTab, setActiveTab] = useState(1)
 
-  const pageLayout = usePageLayout({
-    header: (
+  const Header = () => {
+    return (
       <PageLayout.Header>
         <StackLayout>
           <Text type="heading 5">Systems</Text>
         </StackLayout>
+        <PageLayout.Tabs>
+          <PageLayout.Tab
+            onClick={() => setActiveTab(1)}
+            active={activeTab === 1}
+          >
+            <Icon name="book" size="small" />
+            <Text>Overview</Text>
+          </PageLayout.Tab>
+          <PageLayout.Tab
+            onClick={() => setActiveTab(2)}
+            active={activeTab === 2}
+          >
+            <Icon name="detailed_view" size="small" />
+            <Text>Detailed view</Text>
+          </PageLayout.Tab>
+        </PageLayout.Tabs>
       </PageLayout.Header>
-    ),
+    )
+  }
+  const pageLayout = usePageLayout({
+    header: <Header />,
     footer: <PageLayout.Footer />,
   })
 
+  const renderOverview = () => (
+    <Overview>
+      <ProgressBarWrapper>
+        <ProgressBar
+          data={AllTestKinds}
```

Comment:
These are all mocks, we need real data from model to be imported and used.

Reply:
also, upgrade the Progress bar (but create an issue for that and do it in a separate PR so that Voja can use it as well) so that onClick is optional.

Reply:
Data from model used